src/components/attendee/InputSelect/InputSelect.module.scss

```
@@ -28,5 +32,29 @@
   box-sizing: border-box;
   width: 20%;
   border: none;
-  border-right: 5px solid transparent;
+  border-right: border.$radius--xs solid transparent;
+}
+
+.errorIcon {
+  background-image: url(/assets/icons/input-error-icon.svg);
+  height: $-error-icon-size;
+  width: $-error-icon-size;
+  margin-left: -(space.empty(2));
```

The reason for having `space.empty()` as a function, not a constant:

```suggestion
  margin-left: space.empty(-2);
```

---

[a58b99b](https://github.com/sparkletown/sparkle/pull/2905/commits/a58b99bf83fb6ed1a9b39ec673fb2882dab9e8e9)